python_ta/cfg/visitor.py

```diff
@@ -437,6 +439,64 @@ def visit_with(self, node: nodes.With) -> None:
         for child in node.body:
             child.accept(self)
 
+    def visit_match(self, node: nodes.Match) -> None:
+        """Visit a match statement and create appropriate control flow."""
+        # When only creating cfgs for functions, _current_cfg will only be None outside of functions
+        if self._current_cfg is None:
+            return
+
+        self._current_block.add_statement(node.subject)
+        node.cfg_block = self._current_block
+        after_match_block = self._current_cfg.create_block()
+
+        case_end_blocks = []
+
+        prev_case = self._current_block
+        # Process each match case
+        for case in node.cases:
+
+            edge_label = "No Match" if case_end_blocks else ""
+
+            new_case = astroid.nodes.Name(
```

Comment:
You don't need to create a new node here. It's okay to just store `case.pattern.as_string()`; look at the `_visit` function in `cfg_generator.py` to see how we customize the rendering of certain block types.